lib/util.js

```diff
@@ -222,3 +222,22 @@ exports.parseFormats = function(info, debug) {
   formats.sort(exports.sortFormats);
   return formats;
 };
+
+
+
+/**
+ * @param {String} videoDescriptionHtml
+ * @return {String}
+ */
+exports.cleanVideoDescription = function(videoDescriptionHtml) {
+	if (!videoDescriptionHtml || videoDescriptionHtml == null) 
+		return null;
+
+	return entities.decode(videoDescriptionHtml
+		.replace('\n', ' ', 'gi')
```

Comment:
says [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) flags in this function is deprecated.